package/kernel/mt7621-qtn-rgmii/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=mt7621-qtn-rgmii
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-2.0
+
+PKG_MAINTAINER:=Bjørn Mork <bjorn@mork.no>
+
+include $(INCLUDE_DIR)/package.mk
+
+define KernelPackage/mt7621-qtn-rgmii
+  SECTION:=kernel
+  SUBMENU:=Other modules
+  TITLE:="Enable RGMII connected Quantenna module on MT7621"
```

Comment:
I do not think these quotes are necessary?

Reply:
fixed